src/App.js

```diff
@@ -1748,18 +1646,18 @@ render() {
                   <SendWithLink balance={balance}
                     buttonStyle={buttonStyle}
                     changeAlert={this.changeAlert}
-                    sendWithLink={(amount,cb)=>{
+                    sendWithLink={async (amount, cb)=>{
```

Comment:
It looks like we're not taking advantage of labeling that function `async`, so I also think we shouldn't declare it as such.

Reply:
True, reverted